### ISSUE TITLE
Suggestion: allow empty attribute value when event modifier is used

### ISSUE THREAD

**Issue by HerringtonDarkholme** [vuejs/eslint-plugin-vue#49]
More specifically, only `stop` and `prevent` should be allowed.

**Tell us about your environment**

* **ESLint Version:** 4.0.0
* **eslint-plugin-vue Version:** 3.1.3
* **Node Version:** 8

**Please show your full configuration:**

Vanilla

**What did you do? Please include the actual source code causing the issue.**

```vue

<template>
<el-from @submit.native.prevent></el-form>
</template>
```

**What did you expect to happen?**

No error. Since this is illustrated in the official doc. https://vuejs.org/v2/guide/events.html#Event-Modifiers

**What actually happened? Please include the actual, raw output from ESLint.**

Error.  `'v-on' directives require that attribute value `.
https://github.com/octref/vetur/issues/279

**Comment by KaelWD** [comment]
Can the message be updated to be actual english? I still can't understand what it's trying to say.
